### PR TITLE
access tier group terraform changes 

### PR DESCRIPTION
**Pull request by rohitSangamnerkar**

### CODE REVIEW

File: banyan/resource_service_tunnel.go [hunk 1]
```diff
@@ -354,6 +367,10 @@ func flattenServiceTunnelSpec(d *schema.ResourceData, tun servicetunnel.ServiceT
 		if err != nil {
 			return err
 		}
+		err = d.Set("access_tier_group", nil)
```
Comment (nareshkakubal) [inline banyan/resource_service_tunnel.go:370]: If nil, we need not set it.

File: banyan/resource_service_web.go [hunk 2]
```diff
@@ -280,6 +281,12 @@ func WebSchema() (s map[string]*schema.Schema) {
 				},
 			},
 		},
+		"access_tier_group": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "access tier group which is associated with service",
+			Default:     "",
```
Comment (nareshkakubal) [inline banyan/resource_service_web.go:288]: do we need to set default ?

File: banyan/resource_service_tunnel.go (hunk 1 above)
Comment (rohitSangamnerkar) [reply banyan/resource_service_tunnel.go]: fixed

File: banyan/resource_service_web.go (hunk 2 above)
Comment (rohitSangamnerkar) [reply banyan/resource_service_web.go]: No. Fixed

**Comment by nareshkakubal** [comment]
LGTM

**Review by nareshkakubal — APPROVED** [review]

File: banyan/resource_accesstier_group_test.go [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+func testAccAccessTierGroup_basic_create(name string) string {
+	return fmt.Sprintf(`
+resource "banyan_accesstier_group" "example" {
+	name                   = "%s"
+	description            = "testing-1"
+	cluster                = "cluster1"
+	dns_search_domains     = ""
+	advanced_settings      = "{\"kind\":\"BanyanAccessTierLocalConfig\",\"api_version\":\"rbac.banyanops.com/v1\",\"type\":\"attribute-based\",\"metadata\":{},\"spec\":{\"base\":{\"shield_address\":\"ted-us-west1.shield.bnntest.com:34140\",\"site_address\":\"\"},\"logging\":{},\"events\":{},\"hosted_web_services\":{}}}"
```
Comment (nareshkakubal) [inline banyan/resource_accesstier_group_test.go:37]: @rohitSangamnerkar this is not a right way for advance_settings, we could reuse LocalConfig from accessTier and could be key value pairs. Could you create another PR for the same ?

**Pull request merged**